app/src/foundation/AppProvider.jsx

```diff
@@ -11,14 +11,12 @@ import '@shopify/channels-ui/dist/styles.css';
 import RoutePropagator from './RoutePropagator';
 
 const AppProvider = () => {
-  const {search} = useLocation();
-
   return (
     <ExtendedAppProvider
       polaris={{i18n: polarisTranslations, linkComponent: Link}}
       i18n={translations}
       config={{
-        host: new URLSearchParams(search).get('host'),
+        host: new URL(location).searchParams.get("host"),
```

Comment:
eslint will probably complain about the `"` string
```suggestion
        host: new URL(location).searchParams.get('host'),
```